.github/workflows/ci.yml

```diff
@@ -48,7 +48,6 @@ jobs:
             cxx_flags: ""
 
     runs-on: ubuntu-latest
-    timeout-minutes: 60
```

Comment:
root of all kinds of evil

Reply:
is it possible to stuck without timeout?

Reply:
no because we run each test with `timeout`